src/qml/editorwidgets/DateTime.qml

```diff
@@ -37,20 +37,23 @@ Item {
       Layout.minimumHeight: 48 * dp
 
       Rectangle {
-        anchors.fill: parent
         id: backgroundRect
+        anchors.fill: parent
```

Comment:
Apparently `anchors.fill: label` does not work but:
```suggestion
        height: label.height - label.bottomPadding / 2
        width: label.width
```

Reply:
That doesn't work, rect's height becomes zero.

Reply:
Okay, strange it worked on my system.
With your new approach, it set's the anchors. Beside that it does the same like i suggested, no?

Reply:
Yep, it's seemingly identical, yet on my system it failed to show anything using the alternative suggested. QML can work in mysterious ways ;)

Reply:
> QML can work in mysterious ways

Yes. I experienced that :-)